core-web/libs/sdk/experiments/package.json

```diff
@@ -20,5 +19,13 @@
     "bugs": {
         "url": "https://github.com/dotCMS/core/issues"
     },
-    "homepage": "https://github.com/dotCMS/core/tree/master/core-web/libs/sdk/experiments/README.md"
+    "homepage": "https://github.com/dotCMS/core/tree/master/core-web/libs/sdk/experiments/README.md",
+    "dependencies": {
+        "@jitsu/sdk-js": "^3.1.5"
+    },
+    "files": [
+        "dot-experiments.min.iife.js",
+        "lib/**/*.d.ts"
```

Comment:
How is that we send the iife file to npm?

Reply:
We going to exposed is DotExperiments(the class instance and the client of Jitsu as an attribute), and the iife should go to `dotCMS/src/main/webapp/html/js` to be used by dotCMS VTL, replacing what Rodri did. When I get to the issue related to consuming DotExperiments with Next, I change the entry of this file to the one that should be final.
I use` dot-experiments.min.iife.js` as an entry file because I have a plain HTML demo of dotCMS to simulate all without dotCMS.

Reply:
files and entry are different... here you need to have the file that expose what you will be importing from the npm module, and not the iife which is a bundle.

Reply:
Removed. Not necessary.